simplecpp.cpp

```diff
@@ -3273,14 +3273,14 @@ static std::string getFileIdPath(const std::map<std::string, simplecpp::TokenLis
     }
 
     if (!systemheader) {
-        const std::string relativeOrAbsoluteFilename = getRelativeFileName(sourcefile, header);// unknown if absolute or relative, but always simplified
-        const std::string match = findPathInMapBothRelativeAndAbsolute(filedata, relativeOrAbsoluteFilename);
+        const std::string relativeFilename = getRelativeFileName(sourcefile, header, true);
```

Comment:
@Tal500 I changed this variable name to relativeFilename.. is that ok?

Reply:
Sounds great!